Bottom: 63d3bf9e2dfd8125193799f1932cfb1a832edae3
Top:    775ae69335e53734a2445d133f498759cc3c047c
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-21 14:56:31 +0800

Refresh of virtio-net-optimize-small

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 54085ff..2fe3535 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -687,7 +687,6 @@ static void handle_rx(struct vhost_net *net)
 	vq_log = unlikely(vhost_has_feature(vq, VHOST_F_LOG_ALL)) ?
 		vq->log : NULL;
 	mergeable = vhost_has_feature(vq, VIRTIO_NET_F_MRG_RXBUF);
-	printk("RX!\n");
 
 	while ((sock_len = vhost_net_rx_peek_head_len(net, sock->sk))) {
 		sock_len += sock_hlen;
@@ -714,7 +713,6 @@ static void handle_rx(struct vhost_net *net)
 				vhost_disable_notify(&net->dev, vq);
 				continue;
 			}
-			printk("wait for refill!\n");
 			/* Nothing new?  Wait for eventfd to tell us
 			 * they refilled. */
 			goto out;
